### ISSUE TITLE
Dashboard metrics are cash/realized-only — need mark-to-market across all panels

### ISSUE THREAD

**Issue by allan-mobley-jr** [allan-mobley-jr/gimmes#261]
## Problem

The dashboard treats everything as realized/cash-based when it should be mark-to-market. For a system that holds positions for hours or days, this means most panels show stale or misleading data until positions close.

The position `market_price` is already being refreshed — the data exists, it's just not flowing into the metrics panels.

### Daily P&L

**Current**: Only realized P&L from closed trades today.
```sql
SUM((price - edge) * count) WHERE action = 'close' AND date = today
```
A position up $50 intraday shows $0.00.

**Fix**: `realized_pnl_today + unrealized_pnl_on_open_positions`

### Equity Curve

**Current (no snapshots)**: Cash-only curve from `_equity_curve_from_trades()`. Opens cause a cliff drop, closes cause a jump, nothing in between. Not an equity curve — it's a cash flow chart.

**Current (with snapshots)**: Uses `total_equity` from snapshots, which is itself wrong (#260). Only one data point per cycle (written by Scorecard), so very sparse.

**Fix**: The curve should be `cash + mark_to_market_value` at each data point. Either:
- Write snapshots more frequently (e.g., every SSE refresh cycle), or
- Build the curve from trades + periodic mark-to-market samples, interpolating position value between trade events

### Risk Gauges — Daily Loss

**Current**: Realized-only. Positions getting crushed intraday show 0% on the gauge. The daily loss limit is a circuit breaker that can't see unrealized losses.

**Fix**: `daily_loss = realized_pnl_today + unrealized_pnl`. If this breaches the limit, agents should stop opening new positions.

### Risk Gauges — Largest Position

**Current**: `cost_basis / balance`. Two issues:
1. With bankroll (#258), denominator should be bankroll, not account balance
2. Denominator shifts after purchase — a position sized at exactly 5.0% of $10K shows as 5.2% of $9,505 post-purchase, displaying a false breach

**Fix**: Use bankroll as denominator (after #258). For display, consider using cost_basis at time of sizing, not current balance.

### Performance Metrics (Win Rate, Sharpe, Max Drawdown, Total Return)

**Current**: All derived from either closed-trade counts or the broken equity curve. With only open positions and no closes, everything shows 0.0% / 0.00 / $0.00.

**Fix**: Once the equity curve is mark-to-market, Sharpe / Max Drawdown / Total Return become meaningful even with open positions. Win rate inherently requires closes, which is fine.

## Root cause

The dashboard was built around a realized P&L model. Every metric either:
- Counts closed trades only, or
- Uses cash balance as a proxy for equity

The system already refreshes `market_price` on positions — this needs to flow into Daily P&L, equity curve snapshots, and risk gauge calculations.

## Files

- `src/gimmes/clubhouse/data.py` — `get_portfolio()`, `get_risk()`, `get_metrics()`
- `src/gimmes/reporting/metrics.py` — `calculate_metrics()`, `_equity_curve_from_trades()`, `_apply_equity_curve()`
- `src/gimmes/templates/clubhouse.html` — Daily P&L display
- `src/gimmes/clubhouse/server.py` — consider writing snapshot on each SSE refresh for denser equity curve data

## Related

- #260 — Total Equity calculation is wrong (same root cause)
- #258 — Bankroll replaces session cap (affects risk gauge denominators)
